pipelines/tekton/aiedge-e2e/aiedge-e2e.pipeline.yaml

```diff
@@ -51,9 +51,9 @@ spec:
   - default: $(context.pipelineRun.namespace)
     name: target-namespace
     type: string
-  - name: target-imagerepo
-    type: string
-    description: "The image registry name where the model container for the final image push. Currently supports using quay.io/<target-imagerepo>"
+  - name: target-image-tag-references
+    type: array
+    description: "An array of fully qualified image tag references to be used for the final image push. E.g. registry.example.com/my-org/ai-model:1.0-1"
```

Comment:
This could maybe have a default value of something like:
```
- image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/$(params.model-name):$(params.model-version)-$(context.pipelineRun.uid)
- image-registry.openshift-image-registry.svc:5000/$(context.pipelineRun.namespace)/$(params.model-name):$(params.model-version)-production
```

This could be useful for automated testing, as there wouldn't need to be any registry/credentials specified. Thoughts?

Reply:
Done (well, I've added one of them as a default)